pkg/apis/operator/v1alpha1/tektonchain_defaults_test.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2022 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"context"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/tektoncd/pipeline/test/diff"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/pkg/ptr"
+)
+
+func test_SetDefaults_Chain(t *testing.T) {
```

Comment:
@jkhelil can you camelCase the function name? ==> `testSetDefaultsChain`

error:
```
pkg/apis/operator/v1alpha1/tektonchain_defaults_test.go:29:6: func `test_SetDefaults_Chain` is unused (unused)
func test_SetDefaults_Chain(t *testing.T) {
```